src/contributors/mutations/deleteContributor.ts

```diff
@@ -88,10 +88,33 @@ export default resolver.pipe(
     }
 
     // Delete the notifications related to the project
```

Comment:
```suggestion
    // Disconnect the notifications related to the project
```

Consistency of naming to prevent confusion down the line.